src/pages/bubblegum-v2/burn-cnfts.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Burning Compressed NFTs
+metaTitle: Burning Compressed NFTs | Bubblegum V2
+description: Learn how to burn compressed NFTs on Bubblegum.
+---
+
+The **burnV2** instruction can be used to burn a Compressed NFT and, therefore, remove it from the Bubblegum Tree permanently. To authorize this operation, either the current owner or the delegate authority — if any — must sign the transaction. The instruction accepts the following parameter:
+
+- **Leaf Owner** and **Leaf Delegate**: The current owner of the Compressed NFT and its delegate authority, if any. One of these must sign the transaction.
```

Comment:
Note also if the item is in a collection, then collection must be passed in, which is true for all instructions that have optional collection.  It uses this to check plugins (like making sure the collection isn't frozen, or that Royalties is blocking a transfer), as well as calculating the collection hash for the leaf schema.

There's a couple or few instructions like `delegateV2` which only can be done by the owner, so  they just take a `collection_hash` parameter directly and do not take the collection account.